### PR TITLE
[Quickfix] Fixing comment button misbehaving

### PR DESCRIPTION
**Pull request by ghost**
The comment button places nexto to the field set on large screen. This solves it.

capture d ecran 2015-08-06 a 09 52 39

### CODE REVIEW

**Comment by AugierLe42e** [comment]
Ready to merge.

**Comment by svbergerem** [comment]
`btn-block` sets `display: block` for a button and sets the width to 100%. If we don't want the `display: block` then we shouldn't use `btn-block`. There is however another solution: put the textarea inside a `form-group`:

```diff
diff --git a/app/views/comments/_new_comment.mobile.haml b/app/views/comments/_new_comment.mobile.haml
index 7e8ea3d..bb89a35 100644
--- a/app/views/comments/_new_comment.mobile.haml
+++ b/app/views/comments/_new_comment.mobile.haml
@@ -9,8 +9,7 @@
   - if user_signed_in?
     = form_tag( post_comments_path(post_id), id: "new_comment_on_#{post_id}", class: "new_comment", autocomplete: "off") do
       %fieldset
-        .control-group
-          = hidden_field_tag :post_id,  post_id, id: "post_id_on_#{post_id}"
+        = hidden_field_tag :post_id,  post_id, id: "post_id_on_#{post_id}"
+        .form-group
           = text_area_tag :text, nil, class: "col-md-12 comment_box form-control form-group", id: "comment_text_on_#{post_id}", placeholder: t(".comment")
-          = submit_tag t(".comment"), id: "comment_submit_#{post_id}", 'data-disable-with' => t(".commenting"), class: "btn btn-primary btn-block"
-          .clearfix
+        = submit_tag t(".comment"), id: "comment_submit_#{post_id}", 'data-disable-with' => t(".commenting"), class: "btn btn-primary btn-block"
```

File: app/views/comments/_new_comment.mobile.haml [hunk 1]
```diff
@@ -9,8 +9,7 @@
...
       %fieldset
-        .control-group
-          = hidden_field_tag :post_id,  post_id, id: "post_id_on_#{post_id}"
-          = text_area_tag :text, nil, class: "col-md-12 comment_box form-control form-group", id: "comment_text_on_#{post_id}", placeholder: t(".comment")
-          = submit_tag t(".comment"), id: "comment_submit_#{post_id}", 'data-disable-with' => t(".commenting"), class: "btn btn-primary btn-block"
-          .clearfix
+        = hidden_field_tag :post_id,  post_id, id: "post_id_on_#{post_id}"
+        .form-group.clearfix
+          = text_area_tag :text, nil, class: "col-md-12 comment_box form-control", id: "comment_text_on_#{post_id}", placeholder: t(".comment")
+        = submit_tag t(".comment"), id: "comment_submit_#{post_id}", 'data-disable-with' => t(".commenting"), class: "btn btn-primary btn-block"
```
Comment (jhass) [inline app/views/comments/_new_comment.mobile.haml]: Don't mix hash syntaxes, if you need the rockets (`=>`) use them for all pairs.

**Comment by AugierLe42e** [comment]
Done.

File: app/views/comments/_new_comment.mobile.haml [hunk 2]
```diff
@@ -9,8 +9,7 @@
...
       %fieldset
-        .control-group
-          = hidden_field_tag :post_id,  post_id, id: "post_id_on_#{post_id}"
-          = text_area_tag :text, nil, class: "col-md-12 comment_box form-control form-group", id: "comment_text_on_#{post_id}", placeholder: t(".comment")
-          = submit_tag t(".comment"), id: "comment_submit_#{post_id}", 'data-disable-with' => t(".commenting"), class: "btn btn-primary btn-block"
-          .clearfix
+        = hidden_field_tag :post_id,  post_id, id: "post_id_on_#{post_id}"
+        .form-group.clearfix
+          = text_area_tag :text, nil, class: "col-md-12 comment_box form-control", id: "comment_text_on_#{post_id}", placeholder: t(".comment")
+        = submit_tag t(".comment"), :id => "comment_submit_#{post_id}", 'data-disable-with' => t(".commenting"), :class => "btn btn-primary btn-block"
```
Comment (svbergerem) [inline app/views/comments/_new_comment.mobile.haml]: Please use double quotes.

**Comment by AugierLe42e** [comment]
Ready to merge.

**Comment by svbergerem** [comment]
@AugierLe42e See https://github.com/diaspora/diaspora/pull/6302#issuecomment-128864915. It looks like you edited an outdated version of your code.

**Comment by AugierLe42e** [comment]
Yeah, sorry for that. It should be ok, now.

**Comment by svbergerem** [comment]
Thanks.

**Pull request merged**